nemo/utils/exp_manager.py

```diff
@@ -69,6 +70,15 @@ class CheckpointMisconfigurationError(NeMoBaseException):
     """ Raised when a mismatch between trainer.callbacks and exp_manager occurs"""
 
 
+@dataclass
+class EarlyStoppingParams:
```

Comment:
Add test that checks the dataclass args against PTL EarlyStoppingCallback class. Look at [test inside tests/core/lightning](https://github.com/NVIDIA/NeMo/blob/main/tests/core/test_config_utils.py#L117)